docs/data-management-on-carbondata.md

```diff
@@ -355,6 +355,12 @@ This tutorial is going to introduce all commands and data operations on CarbonDa
   'BAD_RECORDS_ACTION'='REDIRECT','IS_EMPTY_DATA_BAD_RECORD'='false')
   ```
 
+  - **GLOBAL_SORT_PARTITIONS:** If the sort scope is global Sort, user should specify GLOBAL_SORT_PARTITIONS. If user do not specify it, it will use the number of map task. GLOBAL_SORT_PARTITIONS should be Integer type, the range is [1,Integer.MaxValue]，it is only used when the sort scope is global sort.
```

Comment:
I think `the number of map task` is not very precise
@watermen can you provide a more precise one?